src/Runtime/Builtins.hs

```diff
@@ -48,24 +46,25 @@ place = \[v, Vboard arr, Vt [Vi x, Vi y]] -> do
 
 -- | Verifies the parameters are of the expected number & type from their respective lambdas
 -- A final case reports a Runtime error in case anything slips by,
--- which indicates that the number and/or type of parameters didn't match what was expected for that builtin
--- Note: This prevents a crash and gives 'somewhat' better feedback to users, but this should not normally be reachable
+-- which indicates that the number and/or type of parameters didn't match what was expected
+-- for that builtin
+-- Note: This prevents a crash and gives 'somewhat' better feedback to users,
+-- but this should not normally be reachable
```

Comment:
I'm going to add a commit onto this that marks that it was me that left this comment, for future reference.

Reply:
Or in the future, but not so important in this PR actually.